docs/design/autoware-interfaces/ad-api/list/api/fail_safe/index.md

```diff
@@ -1,20 +1,41 @@
+- {{ link_ad_api('/api/fail_safe/mrm/state') }}
 
 ## Description
 
 This API manages the behavior related to the abnormality of the vehicle.
-Provides the state of Request to Intervene (RTI) and Minimum Risk Maneuver (MRM).
+It provides the state of Request to Intervene (RTI), Minimal Risk Maneuver (MRM) and Minimal Risk Condition (MRC).
+As shown below, Autoware has the gate to switch between the command during normal operation and the command during abnormal operation.
+For safety, Autoware switches the operation to MRM when an abnormality is detected.
+Since the required behavior differs depending on the situation, MRM is implemented in various places as a specific mode in a normal module or as an independent module.
+The fail-safe module selects the behavior of MRM according to the abnormality and switches the gate output to that command.
+
+![fail-safe-architecture](./architecture.drawio.svg)
 
 ## States
 
-![fail-safe-state](./state.drawio.svg)
+The MRM state indicates whether MRM is operating. This state also provides success or failure.
+Generally, MRM will switch to another behavior if it fails.
+
+![mrm-state](./mrm-state.drawio.svg)
+
+| State         | Description                                                |
+| ------------- | ---------------------------------------------------------- |
+| NORMAL        | The vehicle is normal. MRM is not operating.               |
+| MRM_OPERATING | MRM is operating because an abnormality has been detected. |
+| MRM_SUCCEEDED | MRM succeeded. The vehicle is in a safe condition.         |
+| MRM_FAILED    | MRM failed. The vehicle is still in an unsafe condition.   |
+
+## Behavior
+
+There is a dependency between MRM behaviors. For example, it switches from a comfortable stop to a sudden stop, but not the other way around.
+This is service dependent. Autoware supports the following transitions by default.
+
+![mrm-behavior](./mrm-behavior.drawio.svg)
 
-| State         | Description                                               |
-| ------------- | --------------------------------------------------------- |
-| NORMAL        | The vehicle is normal.                                    |
-| RTI_OPERATING | The vehicle is normal, but the risk is high.              |
-| MRM_OPERATING | MRM is being executed because an error has been detected. |
-| MRM_SUCCEEDED | MRM succeeded. The vehicle is in a safe condition.        |
-| MRM_FAILED    | MRM failed. The vehicle is still in an unsafe condition.  |
+| State            | Description                                                               |
+| ---------------- | ------------------------------------------------------------------------- |
+| NONE             | MRM is not operating or is operating but no special behavior is required. |
+| COMFORTABLE_STOP | The vehicle will stop quickly with a comfortable deceleration.            |
+| SUDDEN_STOP      | The vehicle will stop immediately with as much deceleration as possible.  |
```

Comment:
@LalithVipulananthan Which do you feel is appropriate? Or other words?

```suggestion
| IMMEDIATE_STOP      | The vehicle will stop immediately with as much deceleration as possible.  |
```

Reply:
I think that `EMERGENCY_STOP` would be more appropriate for this kind of functionality (as suggested by David in Slack).

Reply:
Done https://github.com/autowarefoundation/autoware-documentation/pull/165/commits/a61cbc9ab6c780c0801785891c1f802c75c4c2bb.